tests/testthat/test-hazardlink.R

```diff
@@ -49,3 +49,30 @@ test_that("HazardLinks can merge together", {
     expect_equal(actual@stan@parameters, c("real beta_dsld;", "real beta_ttg;"))
 
 })
+
+
+
+test_that("Priors of the Hazardlink is replaced ", {
+    hazlink<-HazardLink(
```

Comment:
please install `styler` package and style the file or at least the new code section